frontend/components/home/HAboutUs.tsx

```
@@ -0,0 +1,139 @@
+                            <div className='btn-wrap' data-aos='fade-left' data-aos-delay='200'>
+                                <Link href='/about-us' className='btn arrow-btn'>
+                                    Start Free Trial{' '}
+                                    <svg
+                                        width='16'
+                                        height='14'
+                                        viewBox='0 0 16 14'
+                                        fill='none'
+                                        xmlns='http://www.w3.org/2000/svg'
+                                        aria-label='arrow-right'
+                                    >
+                                        <path
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)